content/en/docs/ambient/usage/debug-waypoint/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Debug connectivity issues with waypoints
+description: How to investigate problems routing through waypoint proxies.
+weight: 70
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+This guide describes what to do if you have enrolled a workload in a waypoint proxy, but you are not seeing the expected behavior.
```

Comment:
I recall i had some existing pre-requisites here before debugging waypoints. It is probably good to have them here.

Reply:
Do you remember what they were?  Are you talking about example services you need installed in order to follow these steps?